ci/ci-tests.sh

```diff
@@ -134,8 +134,6 @@ RUSTFLAGS="--cfg=taproot" cargo test --verbose --color always -p lightning
 [ "$CI_MINIMIZE_DISK_USAGE" != "" ] && cargo clean
 RUSTFLAGS="--cfg=splicing" cargo test --verbose --color always -p lightning
 [ "$CI_MINIMIZE_DISK_USAGE" != "" ] && cargo clean
-RUSTFLAGS="--cfg=trampoline" cargo test --verbose --color always -p lightning
```

Comment:
I'm not following the commit message here in 908c6bfb2e1a3099c241f424b7fa72fd47d09d7b since we don't support trampoline forwarding yet?

Reply:
Ok Joost explained that we test that we don't support forwarding basically